src/tile/SpotlightTile.tsx

```diff
@@ -47,12 +51,63 @@ import { useObservableRef } from "../state/useObservable";
 import { useReactiveState } from "../useReactiveState";
 import { useLatest } from "../useLatest";
 
-// Screen share video is always enabled
-const videoEnabledDefault = of(true);
-// Never mirror screen share video
-const mirrorDefault = of(false);
-// Never crop screen share video
-const cropVideoDefault = of(false);
+interface SpotlightItemBaseProps {
+  className?: string;
+  "data-id": string;
+  targetWidth: number;
+  targetHeight: number;
+  video: TrackReferenceOrPlaceholder;
+  member: RoomMember | undefined;
+  unencryptedWarning: boolean;
+  nameTag: string;
+  displayName: string;
+}
+
+interface SpotlightUserMediaItemBaseProps extends SpotlightItemBaseProps {
```

Comment:
Do we use `Base` as to indicate that this is never used anywhere else except as `extends .*Base.*`?
Otherwise I would just remove the `Base` part. If it is just used for extending I like `Base` as a naming schema

Reply:
The name is horribly verbose but… yeah, that is the idea. There's already another interface named `SpotlightUserMediaItemProps`, because to get type safety in React you sometimes have to create a bajillion components (since there's no such thing as calling hooks conditionally… :pensive:)